src/vivarium_public_health/risks/distributions.py

```diff
@@ -230,61 +266,59 @@ def get_exposure_parameters(self, builder: Builder) -> Pipeline:
     # Public methods #
     ##################
 
-    def ppf(self, x: pd.Series) -> pd.Series:
-        exposure = self.exposure(x.index)
+    def ppf(self, quantiles: pd.Series) -> pd.Series:
+        exposure = self.exposure_parameters(quantiles.index)
         sorted_exposures = exposure[self.categories]
         if not np.allclose(1, np.sum(sorted_exposures, axis=1)):
             raise MissingDataError("All exposure data returned as 0.")
         exposure_sum = sorted_exposures.cumsum(axis="columns")
         category_index = pd.concat(
-            [exposure_sum[c] < x for c in exposure_sum.columns], axis=1
+            [exposure_sum[c] < quantiles for c in exposure_sum.columns], axis=1
         ).sum(axis=1)
         return pd.Series(
             np.array(self.categories)[category_index],
             name=self.risk + ".exposure",
-            index=x.index,
+            index=quantiles.index,
         )
 
 
-class DichotomousDistribution(Component):
-    #####################
-    # Lifecycle methods #
-    #####################
+class DichotomousDistribution(RiskExposureDistribution):
+    #################
+    # Setup methods #
+    #################
 
-    def __init__(self, risk: str, _exposure_data: pd.DataFrame):
-        super().__init__()
-        self.risk = EntityString(risk)
+    def build_all_lookup_tables(self, builder: "Builder") -> None:
+        self.lookup_tables["exposure"] = self.build_lookup_table(
```

Comment:
How is it that this was not previously being built?

Reply:
This was previously being built in the Risk component.